### ISSUE TITLE
NPC Indicator respawn timer would be off if the game froze

### ISSUE THREAD

**Issue by senorcookie** [runelite/runelite#13219]
I would sometimes tag-all mobs I'm slaying just to see their respawn locations and timers. If my game froze for whatever reason after the respawn timers were set, the timer would usually be too early in comparison to their real respawn. Un-tagging and then retagging them would fix the problem.

Its quite hard to reproduce the problem I imagine, since you need your game to freeze, but its quite simple to understand how to otherwise replicate the problem lol

I'm using windows 10, RL version 1.7.0, launcher version 2.1.5

**Comment by Nightfirecat** [comment]
I don't think this is a bug we can reasonably solve. If your game is freezing, that is the issue which needs to be fixed, not timers being off after said freezing.
